controllers/patient.controller.js

```diff
@@ -15,9 +16,13 @@ const get = async (req, res) => {
 
 const post = async (req, res) => {
   try {
-    const patient = await patientServices.createPatient(req.body);
+    const value = await patientSchema.validateAsync(req.body);
+    const patient = await patientServices.createPatient(value);
     return res.status(201).json({ id: patient.id });
   } catch (err) {
+    if (err.isJoi) {
+      return res.status(401).json({ error: err.details });
```

Comment:
Why `401`? 401 status code is when user is not authenticated.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status